Cargo.toml

```diff
@@ -44,6 +46,8 @@ pretty_env_logger = "0.5.0"
 dotenvy = "0.15.7"
 rss = "2.0.7"
 arc-swap = "1.7.1"
+rustls = { version = "0.21.12", features = ["dangerous_configuration"] }
```

Comment:
Also this should ideally be gated under the `no-https-verification` feature so that it isn't even enabled unless the `no-https-verification` one is.

Reply:
I think I had a look but I couldn't find the relevant rust docs